src/fragments/lib-v1/auth/android/password_management/10_reset_password_note.mdx

```diff
@@ -0,0 +1,3 @@
+<Callout>
+Note that you must call confirmResetPassword in the same app session as you call resetPassword. If you close the app, you'll need to call resetPassword again.
```

Comment:
is `confirmResetPassword` a method? should this be represented as an inline code block?

```suggestion
Note that you must call `confirmResetPassword` in the same app session as you call `resetPassword`. If you close the app, you'll need to call `resetPassword` again.
```